client/db.go

```diff
@@ -208,6 +209,13 @@ type Store interface {
 
 	// ExecRequest executes the given GQL request against the [Store].
 	ExecRequest(context.Context, string) *RequestResult
+
+	// ACPModule returns the underlying acp module (optional). Incase the access control
+	// is turned off, then the returned acp module will have no value (will be empty).
+	ACPModule(context.Context) (immutable.Option[acp.ACPModule], error)
+
+	// AddPolicy attempts to add policy if the acp module exists, otherwise returns an error.
+	AddPolicy(context.Context, string, string) (AddPolicyResult, error)
```

Comment:
todo: Please document this at least as well as the underlying calls in the ACPModule, if anything the documentation should be more comprehensive here, not the other way around.

Or maybe remove it.

Reply:
Remove what? the documentation here ? or in underlying one?

Reply:
Remove the `AddPolicy` function, it is already on `acp.ACPModule` no?

Reply:
The client calls this function directly, it's clean that way IMO. I can remove this if you and others have a hard preference.

Reply:
The double declaration of some Lens operations in the `LensRegistry` and `Store` caused a bunch of problems, and I wouldn't recommend it.  For example, I've not reviewed the http/cli stuff here yet, but having them both here really should mean you have two endpoints/commands for the same thing there (otherwise you are not sticking to the same client interface and have created multiple different interface structures).

Reply:
> The double declaration of some Lens operations in the `LensRegistry` and `Store` caused a bunch of problems, and I wouldn't recommend it. For example, I've not reviewed the http/cli stuff here yet, but having them both here really should mean you have two endpoints/commands for the same thing there (otherwise you are not sticking to the same client interface and have created multiple different interface structures).

You bring a good point, and I understand that case you mentioned. However I feel uneasy implementing a client side `ACPModule()` function as it would return the underlying acp module (it's a getter).

Note: Currently only `AddPolicy()` has a client side function implementation but not the `ACPModule()`.

Reply:
I do not understand your last comment, there is already a `client.ACPModule()` function.

Reply:
> I do not understand your last comment, there is already a `client.ACPModule()` function.

Yes, but It is not implemented (we are discussing that unimplemented function in another thread so can continue there)

Reply:
Update:
- [x] Removed `ACPModule()` (in commit: ```PR(ACP): Remove `ACPModule` from `client.Store` ```)
- [x] Documented `AddPolicy()` in client/db.go